src/methods/checkDepositWithdrawInfo.ts

```diff
@@ -0,0 +1,31 @@
+import { get } from "lodash";
```

Comment:
I didn't touch much code outside of SEP-6 flow, but I did combine the `checkInfo` methods for SEP-6 and SEP-24 as they are doing the same thing and expect the same payload